src/plugins/intel_cpu/src/transformation_pipeline.cpp

```diff
@@ -611,7 +612,8 @@ void Transformations::MainSnippets(void) {
                                                            ov::is_type<const ov::op::v0::MatMul>(n) ||
                                                            ov::is_type<const ov::op::v1::Transpose>(n) ||
                                                            ov::is_type<const ov::op::v1::Broadcast>(n) ||
-                                                           ov::is_type<const ov::op::v3::Broadcast>(n));
+                                                           ov::is_type<const ov::op::v3::Broadcast>(n) ||
+                                                           (!ngraph::snippets::op::Subgraph::check_broadcast(n)));
```

Comment:
I'm not sure that broadcasting check should be in the plugin callback. I think it should be in [tokenization](https://github.com/openvinotoolkit/openvino/blob/a84f87e9dc7a18c4355c233fae3ef1800c307fff/src/common/snippets/src/pass/collapse_subgraph.cpp#L215) because Snippets doesn't support PDPD broadcasting in general